dylankmoore/BE-MusicStreaming
# SETUP - DBContext, Models, and Program

## Description
- Erased boilerplate code for setup
- Implemented Models
- Implemented folder hierarchy
- Setup DBContext 

## Related Issue
#4  + #3 

## Motivation and Context
- All implemented changes are required for database functionality.

## How Can This Be Tested?
- Pull PR to local testing branch.
- Ensure all models, data, DTO, and api folders are present.
- Ensure DBContext looks correct.

## Screenshots (if appropriate):
N / A

## Types of changes
